lib/builder/src/schema.json

```diff
@@ -45,11 +45,13 @@
     },
     "eslintCacheLocation": {
       "type": "string",
-      "description": "Path to the cache file or directory"
+      "description": "Path to the cache file or directory",
+      "default": ".angular/cache/.eslintcache"
```

Comment:
we could add a new unit test case in builder.spec.ts and check if the default cache file is there after the test run.

Reply:
it worked in the test project for eslint but not stylelint... good idea